EnglishFilter/sections/antiadblock.txt

```diff
@@ -554,6 +554,17 @@ trade2win.com#%#//scriptlet("abort-current-inline-script", "$", "!document.getEl
 @@||encodinghub.com/wp-content/uploads/*.nz/ads/
 ! https://github.com/AdguardTeam/AdguardFilters/issues/72796
 @@||toptechtalk.xyz^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/70882
```

Comment:
Please try to block some requests by `redirect rules`(In addition to these rules, with `important`)